build.gradle.kts

```diff
@@ -0,0 +1,161 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    kotlin("jvm") version "1.6.20"
```

Comment:
This should be unneccesary, and might even be including kotlin's stdlib in the runtime classpath

Reply:
If I remove the Kotlin plugin, Gradle can't parse the build script at all

It does seem like it's including the Kotlin runtime in the jar though, might need to be excluded?

Reply:
Probably need to add `java-library`, that plugin is likely be providing that transitively

Reply:
What would we do without you